### PR TITLE
Copper Golems, Golem Code Fixes

### PR DESCRIPTION
**Pull request by That0nePerson**
## About The Pull Request
Adds a copper golem that's softer and faster than a normal golem. It's also conducive to electricity while being immune to it. Fixes golem IDs, removing the space in them.

## Why It's Good For The Game
Adds in a golem for copper, for some reason there wasn't one before.

## Changelog
:cl:
add: Added a copper golem.
/:cl:

### CODE REVIEW

**Review by Kmc2000 — APPROVED** [review]
This looks alright to me, snake_case is nice to have

File: config/game_options.txt [hunk 1]
```diff
@@ -446,13 +446,13 @@ ROUNDSTART_RACES ipc
...
 ROUNDSTART_RACES apid
 #ROUNDSTART_RACES jelly
-#ROUNDSTART_RACES golem
-#ROUNDSTART_RACES adamantine
-#ROUNDSTART_RACES plasma
-#ROUNDSTART_RACES diamond
-#ROUNDSTART_RACES gold
-#ROUNDSTART_RACES silver
-#ROUNDSTART_RACES uranium
+#ROUNDSTART_RACES iron_golem
```
Comment (ike709) [inline config/game_options.txt:449]: You need to edit the game options for Sage as well.

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 2]
```diff
@@ -179,11 +180,24 @@
 /datum/species/golem/silver/on_species_loss(mob/living/carbon/C)
 	REMOVE_TRAIT(C, TRAIT_HOLY, SPECIES_TRAIT)
 	..()
+// Softer and faster, but conductive
+/datum/species/golem/copper
+	name = "Copper Golem"
+	id = "copper_golem"
+	fixed_mut_color = "#d95802"
+	speedmod = 1.5
```
Comment (ike709) [inline code/modules/mob/living/carbon/human/species_types/golems.dm:188]: This is actually slower than about half of the golems. Lower speedmod is faster, higher is slower.

**Comment by That0nePerson** [comment]
DIdn't realize I had to change sage game options as well, done

File: config/game_options.txt (hunk 1 above)
Comment (That0nePerson) [reply config/game_options.txt:449]: done, thanks for reminding me

**Review by ike709 — APPROVED** [review]

**Comment by ike709** [comment]
@Crossedfall Config

**Comment by qwertyquerty** [comment]
@Crossedfall figz bro

**Comment by qwertyquerty** [comment]
@That0nePerson wiki changes bro

**Comment by That0nePerson** [comment]
@qwertyquerty sorry, wiki changes made on the golems page. Didn't see anywhere else that needed a change

**Review by Crossedfall — APPROVED** [review]
Config set

**Pull request merged**